src/workerd/io/worker.h

```diff
@@ -638,10 +638,18 @@ class Worker::Actor final: public kj::Refcounted {
 
   using Id = kj::OneOf<kj::Own<ActorIdFactory::ActorId>, kj::String>;
 
+  class Loopback {
```

Comment:
bikeshedding: Is this really a loopback? It kinda feels like a factory type for a specific actor.